ext/RMagick/rmutil.c

```diff
@@ -1468,6 +1469,9 @@ rm_progress_monitor(
     RB_GC_GUARD(span);
 
     return RTEST(rval) ? MagickTrue : MagickFalse;
+#else
+    return MagickTrue;
```

Comment:
Does this represent a success? If so, would it make more sense to return `MagickFalse` here to indicate to users that it doesn't work? I'm also wondering if it would be better to raise an error instead, though maybe that would be more risky.

Reply:
If return `MagickFalse` in this callback, monitoring process will be stopped in ImageMagick. So we have to return `MagickTrue`